.pre-commit-config.yaml

```diff
@@ -5,8 +5,4 @@ repos:
     - id: flake8
       args: # arguments to configure flake8
         # making isort line length compatible with autopep8
-        - "--max-line-length=88"
--   repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: 'v1.5.7'
-    hooks:
-    - id: autopep8
```

Comment:
why was this part removed?

Reply:
I later learnt that flake8 lets us know if there is an error. Whereas autopep8 automatically makes the changes. Wanted the former than the later.